src/d3a/models/market/two_sided.py

```diff
@@ -254,18 +228,23 @@ def accept_bid_offer_pair(self, bid, offer, clearing_rate, trade_bid_info, selec
         return bid_trade, trade
 
     def match_offers_bids(self):
```

Comment:
This method can be removed from the markets now.

Reply:
I kept it for redis market communication, since we haven't used area markets communication via redis, I assume we can safely deprecate it ?